src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - 50;
+  }
+
+  if (days >= 3) {
+    return total - 20;
+  }
+
+  if (days < 3) {
+    return total;
+  }
```

Comment:
on my opinion, you dont need if statement here, because if previus condition are false, you anyway will return total